tools/tpm2_verifysignature.c

```diff
@@ -153,7 +154,10 @@ static bool init(TSS2_SYS_CONTEXT *sapi_context) {
     }
 
     if (ctx.flags.sig) {
-        bool res =  files_load_signature(ctx.sig_file_path, &ctx.signature);
+
+        TPMI_ALG_SIG_SCHEME sig_alg = ctx.flags.raw ? TPM2_ALG_RSASSA : TPM2_ALG_ERROR;
+        tpm2_convert_sig_fmt fmt = ctx.flags.raw ? signature_format_plain : signature_format_tss;
+        bool res = tpm2_convert_sig_load(ctx.sig_file_path, fmt, sig_alg, ctx.halg, &ctx.signature);
```

Comment:
Ensure that ctx.halg is specified with -r.

Should we change -r to -f format and handle what they pass as an argument.

Reply:
@williamcroberts, i think so too. Also may be we should call the fmt variable signature_format_ssl_rsa instead of signature_format_plain